README.md

```diff
@@ -1,68 +1,35 @@
 <head>
-  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Serverless Cache">
+  <meta name="Momento JavaScript Client Library Documentation" content="JavaScript client software development kit for Momento Serverless Cache">
 </head>
 <img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
 
 [![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
-[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md) 
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
 
-# Momento Node.js Client Library
+# Momento JavaScript Client Libraries
 
+* Website: [https://www.gomomento.com/](https://www.gomomento.com/)
+* Documentation: [https://docs.momentohq.com/](https://docs.momentohq.com/)
+* Getting Started: [https://docs.momentohq.com/getting-started](https://docs.momentohq.com/getting-started)
+* Discuss: [Momento Discord](https://discord.gg/3HkAKjUZGq)
 
-Node.js client SDK for Momento Serverless Cache: a fast, simple, pay-as-you-go caching solution without
-any of the operational overhead required by traditional caching solutions!
+Momento Serverless Cache is a fast, simple, pay-as-you-go caching solution without any of the operational overhead
```

Comment:
We've dropped "Serverless" from this. We need to update it to just "Momento Cache"

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/529/commits/7772696a548e62a8805d752f9f95e5e6ac993644